display_centre/display.py

```diff
@@ -637,10 +635,15 @@ def hex_from_rgb(r, g, b):
 
     def _update_screen_every_second(self):
         self.refresh_display()
-        self.tk.after(500, self._update_screen_every_second)
+        self.tk.after(50, self._update_screen_every_second)
 
+    last_refreshed = 0
+    REFRESH_LIMIT = 100
     def refresh_display(self):
         if self.data.update_screen:
```

Comment:
can you just quickly explain the idea behind this ?

Reply:
so what it should do (and seems to unless you've noticed otherwise?) is cap the display refreshes at once every 100ms - faster than the previous refresh speed but still seems performant :)